content/applications/essentials/activities.rst

```diff
@@ -71,17 +73,16 @@ List view
 
 Activities can also be created from a list view.
 
-If the :guilabel:`Activities` column is hidden, reveal it using the :guilabel:`(additional options
-toggle)` icon in the far-right of the top row, represented by two horizontal lines with two dots.
+If the :guilabel:`Activities` column is hidden, reveal it using the :icon:`oi-settings-adjust`
+:guilabel:`(adjust)` icon in the far-right of the top row.
```

Comment:
I fully understand the use of (adjust) here as the icon descriptor, however, in the past, I *believe* we *may* have used 'additional options' as the name of the icon -- but not sure if that's where we landed **officially** -- either way, might be worth checking with Sam about this one during your revisions to see what he thinks is the best route... 👍

Reply:
I checked with @samueljlieber, we are supposed to use the official name (from dev tools), at least for now, just to be consistent. :+1: